targets/UnrealMarketplacePlugin/examplesource/ExampleProject/Source/ExampleProject.Target.cs

```diff
@@ -9,6 +9,7 @@ public class ExampleProjectTarget : TargetRules
 {
     public ExampleProjectTarget(TargetInfo Target) : base(Target)
     {
+        bUseUnityBuild = false;
```

Comment:
This should be for testing only ? We should remove that from the example project before submission. Also, did you test it with the Unity build turned on ?

Reply:
It's possible in a future checkin to set that flag for the Example Project, and not for the plugin.
No, I only tested it as written here.